jme3-core/src/test/java/com/jme3/math/QuaternionTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2023 jMonkeyEngine
```

Comment:
It's 2024, not 2023.

Reply:
Well that's embarrassing. That isn't a copy-paste fail. I removed the year from the one I got the copyright notice from, thought "what year is it?" and wrote 2023. Anyway I've corrected that and the other issue

Reply:
Thanks for taking care of things.